README.md

```diff
@@ -45,3 +45,54 @@ As seen in the flowchart previously, the drone ignores the contents of the routi
 ### Nacks
 When sending a Nack, that contains a packet, the packet inside will have its hop index pointing to where the drone would have sent that packet if it wasn't for the nack.
 
+## List of panics
+#### "The Sender<DroneCommand> end of the simulation controller channel unexpectedly got dropped");
+ - There is no link anymore with the simulation controller
+
+#### "There is no connected sender to the drone's receiver channel and no DroneCommand::Crash has been received"
+ - Trigger only if all the channels have been deleted even the controller
+
+#### "Tried to set an invalid pdr value of {pdr}, which is not in range (0.0..=1.0)"
+ - Trigger if you try to set an incorrect new value of the pdr
+
+#### "Cannot add a channel with the same NodeId of this drone (which is {})"
+ - You cannot connect the drone to itself or another drone with the same id
+
+#### "Cannot remove channel to {node_id}: it does not exist"
+ - You try to remove a channel that already doesn't exist
+
+#### "not expecting a packet of type flood request"
+ - Not suppose to be trigger, call of the `process_not_flood_request` function with a flood request
+
+#### "empty routing header for packet {packet}"
+ - The packet pass in function `process_not_flood_request` as an empty source routing header
+
+#### "hop_index out of bounds: index {current_index} for hops {:?}"
+ - The hop index in the packet pass in function `process_not_flood_request` 
+is out of bounds of the **Vec** of hops.  
+
+#### "received packet with hop_index 0, which should be impossible"
+ - The packet received as an hope index of 0 in `process_not_flood_request`, this is impossible in normal situation.
+Maybe this packet was suppose to be a **flood request**
+
+#### "expecting a packet of type flood request"
+ - Not suppose to be trigger, call of the `process_flood_request` function with a packet that is not a flood request
+
+#### "flood request has no path trace"
+ - Maybe we should'nt keep this one but that indicate that another drone or server
+sent an flood request with an empty path trace. He should at least had himself to the path
+
+#### "next hop not found"
+ - Trigger when the drone try to send a packet <send packet function> to the next hop but we can't find it.
+Maybe the next hop is not connected or he not exist
+
```

Comment:
Given that this panic should not happen unless there is some error on our part I wouldn't document at all, I just added 7854271 so that the panic is a bit more clear, but the most we should add here is that they should contact us because it should not happen

Reply:
ok I can add something like `if you encouter any other panic from our drone contact us`